### ISSUE TITLE
Windows Protection & Multi-Task

### ISSUE THREAD

**Issue by TheDragonSlays** [giabao4498/Hydra-Virus#12]
Hi, I've noticed that if you install it onto a computer with a decent amount of Windows Virus Protection it will auto delete the .exe off of the computer (after it gives you a virus found popup), which means that it is running enough commands to edit the system and let it work that it flags it. This is probably because it is not a verified as a safe application on the windows servers.

Also can you add a feature where in task manager instead of it saying that it is multiple instances of one program make it be an individual thing, this way if you end it in task manager more appear anyways this way the only way to end it is by shutting down/restarting the computer. You may already have this feature implemented and I just don't have the version that does it but either way this will increase the difficulty of stopping it. I have not found a way to do this myself but if I do I will link it.

**Comment by giabao4498** [comment]
Yeah, starting new virus programs instead of creating new instances of the same program is harder to do. Let me know if you figure out how to do it 😄
